app/controllers/RegularContributions.scala

```diff
@@ -36,8 +36,10 @@ class RegularContributions(
 
   def displayForm(paypal: Option[Boolean]): Action[AnyContent] = AuthenticatedAction.async { implicit request =>
     identityService.getUser(request.user).semiflatMap { fullUser =>
-      isMonthlyContributor(request.user.credentials).recover({ case _ => None }) map {
-        case Some(true) => Redirect("/contribute/recurring/existing")
+      isMonthlyContributor(request.user.credentials) map {
+        case Some(true) =>
+          SafeLogger.info(s"Determined that ${request.user.id} is already a monthly contributor; re-directing to /contribute/recurring/existing")
```

Comment:
I thought that generally we were trying to avoid logging personally identifiable information for GDPR purposes. @davidfurey do you have an opinion about this?

Reply:
I discussed this with @davidfurey and @paulbrown1982. I think the general consensus was that PII in Cloudwatch logs will be acceptable for our purposes (as long as the log groups are encrypted and have a retention policy of 14 days). We have a valid reason for storing small amounts of PII (e.g. identity ids) in these detailed logs, as it enables us to debug issues effectively.

However, we don't want this PII leaking into other systems, and I've been doing [some work](https://github.com/guardian/support-frontend/pull/494) to ensure that these type of log messages are scrubbed before being sent on to Sentry.

Reply:
other systems == systems with storage not in our direct control - typically SaaS 3rd parties. PaaS 3rd parties such as AWS, ExactTarget, Salesforce we should configure our own storage and retention policies as Jacob mentioned.

Reply:
👍